ci-operator/config/openshift/svt/openshift-svt-master__reliability-v2-aws-4.18-nightly-x86.yaml

```diff
@@ -0,0 +1,72 @@
+base_images:
+  upi-installer:
+    name: "4.18"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.18"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- always_run: false
+  as: reliability-v2-install-only
+  steps:
+    cluster_profile: aws-perfscale-lrc-qe
+    env:
+      BASE_DOMAIN: qe-lrc.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "3"
+      COMPUTE_NODE_TYPE: m5.xlarge
+      REMOTE_CLIENT: "true"
+    pre:
+    - chain: ipi-aws-pre
+    test:
+    - ref: idp-htpasswd
+    - chain: openshift-svt-reliability-v2
+- always_run: false
+  as: reliability-v2-20h
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "3"
+      COMPUTE_NODE_TYPE: m5.xlarge
+      PROW_CLUSTER_TYPE: self-managed
+      RELIABILITY_DURATION: 20h
```

Comment:
@qiliRedHat will this change allow us to run a reliability test for 7 days? I wonder because I see two scenarios, in one place `RELIABILITY_DURATION` is set to 20h, and in another place, it's 1h.

Reply:
@mehabhalodiya No, we can not run reliability on Prow for 7 days as there is a hard limitation of 24 hours. So except the installation, pre and post, the longest time we can run is about 20 hours. The 1h job is for a quick test.